### PR TITLE
Reload session after webview background termination

### PR DESCRIPTION
**Pull request by scruff311**
Starting in iOS 15, the OS will terminate webviews if it needs memory for other purposes. The result is a blank webview when the app is brought back to the foreground. Navigating to another tab will reload the page if the user's session is still valid. This PR will reload the session as soon as the OS tries to terminate it so that the content remains on screen when the app is brought back to the foreground.

More information on the blank webview bug can be found at https://blog.embrace.io/bug-of-the-month-blank-webviews

### CODE REVIEW

**Review by jpcamara — DISMISSED** [review]

File: Turbolinks/Visit.swift [hunk 1]
```diff
@@ -15,6 +15,8 @@ protocol VisitDelegate: AnyObject {
     func visitRequestDidStart(_ visit: Visit)
     func visit(_ visit: Visit, requestDidFailWithError error: NSError)
     func visitRequestDidFinish(_ visit: Visit)
+    
+    func visitDidTerminate(_ visit: Visit)
```
Comment (zhanfengh) [inline Turbolinks/Visit.swift:19]: The name `visitDidTerminate` feels a bit confusing. Maybe something like `webViewprocessDidTerminate`?

On the other hand, I do see the pattern of having a prefix of `visit` for all the functions here.

🤔

**Review by zhanfengh — COMMENTED** [review]
Just one quick question on the function name, otherwise, looking great!

File: Turbolinks/Visit.swift (hunk 1 above)
Comment (scruff311) [reply Turbolinks/Visit.swift:19]: yea, the convention with all of the other protocol methods is why i stuck with `visit`

**Review by zhanfengh — APPROVED** [review]
👏

**Review by treiff — APPROVED** [review]
👍

**Pull request merged**